client/temp/v5AlternativePaymentMethods/bancontact/html/src/app.js

```diff
@@ -0,0 +1,116 @@
+const v6Button = document.querySelector("#v6-paypal-button");
+const v5ButtonContainer = document.querySelector("#v5-bancontact-button");
+let showV6Button;
+let showV5Button;
+
+function showButtons() {
```

Comment:
Nit: for readability, can we have these be boolean variables and remove the `hidden` attr directly in this function? Ex:
```js
function showButtons() {
  if (shouldShowV6Button && shouldShowV5Button) {
    v6Button.removeAttribute("hidden"); 
    v5ButtonContainer.removeAttribute("hidden");   
  }
}
```